src/PIL/JxlImagePlugin.py

```diff
@@ -0,0 +1,180 @@
+from __future__ import annotations
+
+import struct
+from io import BytesIO
+
+from . import Image, ImageFile
+
+try:
+    from . import _jxl
+
+    SUPPORTED = True
+except ImportError:
+    SUPPORTED = False
+
+
+## Future idea:
+## it's not known how many frames does animated image have
+## by default, _jxl_decoder_new will iterate over all frames without decoding them
+## then libjxl decoder is rewinded and we're ready to decode frame by frame
+## if OPEN_COUNTS_FRAMES is False, n_frames will be None until the last frame is decoded
+## it only applies to animated jpeg xl images
+# OPEN_COUNTS_FRAMES = True
+
+
+def _accept(prefix):
+    is_jxl = (
+        prefix[:2] == b"\xff\x0a"
+        or prefix[:12] == b"\x00\x00\x00\x0c\x4a\x58\x4c\x20\x0d\x0a\x87\x0a"
+    )
+    if is_jxl and not SUPPORTED:
+        return "image file could not be identified because JXL support not installed"
```

Comment:
Exception instead of return string?

Reply:
Should we rise `SyntaxError` like [here](https://github.com/python-pillow/Pillow/blob/b3edfb08ce274f3e0b177b92d317f3d3d0a85248/src/PIL/Jpeg2KImagePlugin.py#L196) in Jpeg2K plugin?
I based my work on WebP plugin which [just returns string](https://github.com/python-pillow/Pillow/blob/b3edfb08ce274f3e0b177b92d317f3d3d0a85248/src/PIL/WebPImagePlugin.py#L34).